go/capture/adapter.go

```diff
@@ -27,6 +27,8 @@ func CaptureResponseChannel(stream pc.Driver_CaptureClient) <-chan CaptureRespon
 	var ch = make(chan CaptureResponse, 4)
 	go func() {
 		for {
+			// Use Recv because ownership of |m| is transferred to |ch|,
```

Comment:
Just curious, is there any anti-patterns when you add this comment?

Reply:
I don't quite follow (anti-patterns about what?).

What this comment is getting at is that gRPC provides another RecvMsg call, which takes an owned instance and fills into it. Often that'll be used because it avoids an allocation. This comment was coped from go/materialize/adapater.go to note that that's not allowed here given the ownership semantics. In practice, it also doesn't help because we're using an adapter channel which wants to returned owned instances anyway.

Reply:
got it. Thanks!